fenix/app/src/main/java/org/mozilla/fenix/onboarding/JunoOnboardingFragment.kt

```diff
@@ -219,10 +220,12 @@ class JunoOnboardingFragment : Fragment() {
     private fun isNotATablet() = !resources.getBoolean(R.bool.tablet)
 
     private fun pagesToDisplay(
+        context: Context,
         showNotificationPage: Boolean,
         showAddWidgetPage: Boolean,
     ): List<OnboardingPageUiData> =
         FxNimbus.features.junoOnboarding.value().cards.values.toPageUiData(
+            context,
```

Comment:
❓ Do we need to pass in context here or would `jexlHelper` suffice?

In general, when we keep the android framework outside of mapping logic, it makes it possible to write unit tests that run on jvm without using roboelectric that makes the tests slower. It's also a good abstraction to keep android framework at the edge of the system/feature, in this case the fragment and UI/composables.